contracts/eden/src/actions/elect.cpp

```diff
@@ -39,7 +39,7 @@ namespace eden
       }
       else
       {
-         eosio::check(member.election_participation_status() == in_election,
+         eosio::check(member.election_participation_status() != not_in_election,
```

Comment:
This check and the one above don't match the new definition of whether the user is opted in